exporter/signalfxexporter/translation/converter.go

```diff
@@ -152,6 +154,60 @@ func labelsToDimensions(labels pdata.StringMap, extraDims []*sfxpb.Dimension) []
 	return dimensions
 }
 
+func convertSummaryDataPoints(
+	in pdata.SummaryDataPointSlice,
+	name string,
+	extraDims []*sfxpb.Dimension,
+) []*sfxpb.DataPoint {
+	out := make([]*sfxpb.DataPoint, 0, in.Len())
+
+	for i := 0; i < in.Len(); i++ {
+		inDp := in.At(i)
+
+		dims := labelsToDimensions(inDp.LabelsMap(), extraDims)
+		ts := timestampToSignalFx(inDp.Timestamp())
+
+		countPt := sfxpb.DataPoint{
+			Metric:     name + "_count",
+			Timestamp:  ts,
+			Dimensions: dims,
+			MetricType: &sfxMetricTypeCumulativeCounter,
+		}
+		c := int64(inDp.Count())
+		countPt.Value.IntValue = &c
+		out = append(out, &countPt)
+
+		sumPt := sfxpb.DataPoint{
+			Metric:     name,
+			Timestamp:  ts,
+			Dimensions: dims,
+			MetricType: &sfxMetricTypeCumulativeCounter,
+		}
+		sum := inDp.Sum()
+		sumPt.Value.DoubleValue = &sum
+		out = append(out, &sumPt)
+
+		qvs := inDp.QuantileValues()
+		for j := 0; j < qvs.Len(); j++ {
+			qPt := sfxpb.DataPoint{
+				Metric:     name + "_quantile",
+				Timestamp:  ts,
+				MetricType: &sfxMetricTypeGauge,
+			}
+			qv := qvs.At(j)
+			qdim := sfxpb.Dimension{
+				Key:   "quantile",
+				Value: strconv.FormatFloat(qv.Quantile(), 'f', 6, 64),
```

Comment:
Six digits of precision seems a little high to me for quantile in most cases ("0.250000" when "0.25" is likely desired).  Does TrimRight(quantile, "0") make sense?

Reply:
Hmm, I'm not sure. This line was copied from SA -- figured we'd want to keep it the same
https://github.com/signalfx/signalfx-agent/blob/c295133b4a52b2fff071c8070536183a48e403fc/pkg/monitors/prometheusexporter/conversion.go#L77

Reply:
I'm not sure there would be any compatibility guarantees from this monitor to the exporter, but can see it either way.

Reply:
Apologies, I didn't mean to dismiss your idea -- I'm partial to it. @keitwb do you know where the six digits of precision came from?

Reply:
Seems like `-1` would be the best option for the `prec` argument.  That makes it use the least number of decimal points required.  I can't remember why -6 was used, might have been no reason at all.

Reply:
Thank you, done.